index.js

```diff
@@ -216,9 +224,13 @@ function serialize (obj) {
 }
 
 function readError (reader) {
-  const rejectionReason = readIlpError(reader)
+  const code = reader.read(3).toString('ascii')
+  const name = reader.readVarOctetString().toString('ascii')
+  const triggeredAt = readGeneralizedTime(reader)
+  const data = reader.readVarOctetString().toString('utf8')
```

Comment:
according to https://github.com/interledger/rfcs/pull/291/files#diff-2c87a4d4e57dd9430f40e0fe2b4d295aR66 it can be any data, so this may fail if it the bytes are not interpretable as correct utf8, right?